.git-blame-ignore-revs

```diff
@@ -0,0 +1,10 @@
+# git 2.23+ (released in August 2019)
+#
+# $ git config blame.ignoreRevsFile .git-blame-ignore-revs
```

Comment:
I guess the git blame ignore ref feature isn't well-known yet amongst devs. Like to include a comment like
```
See https://michaelheap.com/git-ignore-rev/
```
?

Reply:
I would prefer a word about man page.

Reply:
Also fine, of course. Also less prone to dead links ;-)

Reply:
Yep, also I though that it was pretty easy to understand from the name :)